pickLayer/plugin.py

```diff
@@ -124,7 +124,7 @@ def initGui(self) -> None:  # noqa N802
             parent=iface.mainWindow(),
             set_checkable=True,
             add_to_toolbar=True,
-            add_keyboard_shortcut=True,
+            keyboard_shortcut="Shift+P",
```

Comment:
I think it would be good idea to have keyboard shortcuts in one place, eg in enum
```
class KeyboardShortcut(Enum):
    SELECT_ACTIVE_LAYER = "Ctrl+P"
```

Reply:
Created enum for shortcuts to settings.py